src/main/java/com/kryptnostic/conductor/rpc/ConductorElasticsearchApi.java

```diff
@@ -39,6 +47,8 @@
 
 	Boolean initializeEntitySetDataModelIndex();
```

Comment:
because the return type needs to be used by the lambdas for Function<ConductorSparkApi, Boolean>